Gemfile.lock

```diff
@@ -163,3 +163,6 @@ DEPENDENCIES
   turbolinks
   uglifier (>= 1.3.0)
   web-console (~> 2.0)
+
+BUNDLED WITH
```

Comment:
This is auto-generated when we run 'bundle install' I believe so we don't have to worry about it.